src/renderer/vt/state.cpp

```diff
@@ -246,18 +246,41 @@ CATCH_RETURN();
 [[nodiscard]] HRESULT VtEngine::UpdateViewport(const SMALL_RECT srNewViewport) noexcept
 {
     auto hr = S_OK;
-    const auto oldView = _lastViewport;
     const auto newView = Viewport::FromInclusive(srNewViewport);
+    const auto oldSize = _lastViewport.Dimensions();
+    const auto newSize = newView.Dimensions();
 
-    _lastViewport = newView;
-
-    if ((oldView.Height() != newView.Height()) || (oldView.Width() != newView.Width()))
+    if (oldSize != newSize)

```

Comment:
I was testing whether `VtEngine::UpdateViewport` works properly with the new, additional call to `UpdateViewport` on the first render cycle. It works correctly, but it now called `InvalidateAll()` during the start.

These changes here fix the issue by moving the entire `if (_resizeQuirk)` block into the size check. Additionally I made a slight change to the code to move from comparing `Viewport`s to comparing `COORD`s, which simply the code.

Before:
```cpp
if ((oldView.Height() != newView.Height()) || (oldView.Width() != newView.Width()))
```

After:
```cpp
if (oldSize != newSize)
```